README.md

```diff
@@ -1,8 +1,45 @@
+
+## Skills
+
+**Proficient:** Java, Linux/Bash, RDBMS/SQL, Agile/Scrum, Git/Github
+
+**Competent:** Docker, ReactJS/GatsbyJS/NextJS, AngularJS, Python, Android
+
+**Familiar:** C programming language, Numpy, UX/UI design, Advanced Object Oriented Programming, Kotlin, Pandas
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_HunainKhalid](https://github.com/jarviscanada/jarvis_data_eng_HunainKhalid)
+
+
+**Linux Cluster Resource Monitoring App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HunainKhalid/tree/master/linux_sql)]: Developed a linux cluster monitoring agent for the Jarvis Cluster Team (JCT).The application sends reports of server usage to the JCT. Reports determine whether to add or remove servers. Implemented in CentOS7, using a Docker container with an attached PSQL instance and IntelliJ's bash script extension. Scripts and SQL code are sent to Jarvis remote servers. Once sent, monitoring of database insertions was done in real-time through database viewer, DBeaver. Testing was done manually. All code is deployed to the git url above.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HunainKhalid/tree/master/core_java)]:
+      
+  - Grep App: Developed a Java application simulating Linux (grep) functionality. A Linux command that allows searching of various text-patterns in files or files themselves. Implemented using Java libraries FileReader, Filewriter, BufferedReader,BufferedWriter and Lanbda expressions. Built with Maven in Intellij's platform. This project resides in a docker container, pushed to Docker Hub.
+  - 
+  - 
+
+
+## Highlighted Projects
+**Portfolio(Personal) Website**: Created a personal portfolio (hunain.dev) using next.js and tailwind.js. Deployed on Vercel for hybrid static site generation, improved loading times and safety. Continuous integration, improvements and  version control are done from Git and Vercel.
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis Consultancy Group (2021-present)**: Worked as a software developer developing projects on Linux CentOS7 using SQL, RDBMS, Docker, Maven and Github, in an agile environment. Actively participated in daily scrum meetings, often collaborating and helping each other to find soluions. Code review sessions and sprint retro meetings with senior developers helped me understand the agile/scrum developmentcycle further.
```

Comment:
**Software Developer, Jarvis Consultancy Group (2021-present)**: Worked as a software developer developing projects using Java, Linux, RDBMS/SQL, Docker, Maven, Github, etc. in an agile environment. Actively participated in daily scrum meetings, often collaborating and helping team members solve problems in innovative ways. Organized code review sessions and scrum meetings to deliver high-quality work.